### DISCUSSION TITLE
Evolve should ignore or change behavior when Image Number != 1

### DISCUSSION THREAD

**Discussion by halr9000** [runew0lf/RuinedFooocus#106] [Ideas]
Had never used evolve before, and I had image number set to 4. Took me a bit to realize that those numerals from 1-9 were in fact supposed to be a 3x3 grid. Maybe you could ignore image number? Also, it's not clear to me what happens when I click the numbers. Is this written up anywhere how it works? Could not find anything in the wiki.

**Comment by yownas** [comment]
It was initially just a quick hack. Whenever we get around to update the documentation I'll try to explain it in a good way. :) But basically what it does is that it will take the prompt, create 9 versions of it where it randomly change words (or CLIP tokens), you can den select the number that correspond to the image you like best and create new versions of it.

...and, yes, it really should ignore the image number, and force it to 1. I'll put that on the list of things to fix.
